2.8.4.2 Changelog (#5756)
Kong/docs.konghq.com: app/gateway/changelog.md

* Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
* Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21

> ## 2.8.4.2

Reviewer: @acgoldsmith

Shall we add the two lines at https://github.com/Kong/kong-ee/blob/next/2.8.x.x/CHANGELOG-EE.md?plain=1#L11 to this PR?

* Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
* Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21

## 2.8.4.2

**Release Date** 2023/06/30

### Fixes
* Fixed a bug where subrequests produced by the `error_page` directive could interfere with worker processes when *buffered proxying* is enabled.

#### Kong Manager
* Fixed an issue where the Zipkin plugin didn’t allow the addition of `static_tags` through the Kong Manager UI. 
* Fixed an issue where some of the icons were not rendering correctly.

#### Plugins
* Fixed an issue with the Oauth 2.0 Introspection plugin where a request with JSON that is not a table failed.
* Fixed an issue where the slow startup of the Go plugin server caused a deadlock.

> * Fixed an internal error that occurred while validating the schema if `http_endpoint` contained userinfo but the headers were empty.

Reviewer: Probably, we should remove this line. I cannot locate the commit in `2.8.4.2-rc1`.

```suggestion
```

* Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
* Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21

## 2.8.4.2

> **Release Date** 2023/06/30

Author: ```suggestion
**Release Date** 2023/07/07
```

### Fixes

> * Fixed a bug where subrequests produced by the `error_page` directive could interfere with worker processes when *buffered proxying* is enabled.

Author: ```suggestion
* Fixed a bug where internal redirects, such as those produced by the `error_page` directive,
  could interfere with worker process handling the request when *buffered proxying* is
  being used.
```
